app/code/Magento/Inventory/Test/Unit/Model/SourceTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Test\Unit\Model;
+
+use \Magento\Inventory\Model\Source;
+
+class SourceTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * constants for testing value purpose
+     */
+    const TEST_ID = 1;
+    const TEST_STRING = 'Jxecqi_Ahuytudxkruh';
+    const TEST_IS_ACTIVE = 1;
+
+    /**
+     * @var \Magento\Framework\TestFramework\Unit\Helper\ObjectManager
+     */
+    protected $objectManager;
+
+    /**
+     * @var Source
+     */
+    protected $source;
+
+    protected function setUp()
+    {
+        $this->objectManager = new \Magento\Framework\TestFramework\Unit\Helper\ObjectManager($this);
+        $this->source = $this->objectManager->getObject(Source::class);
+    }
+
+
+    public function testSourceId()
+    {
+        $this->source->setSourceId(SourceTest::TEST_ID);
+        $this->assertEquals($this->source->getSourceId(), SourceTest::TEST_ID);
+    }
+
+    public function testName()
+    {
+        $this->source->setName(SourceTest::TEST_STRING);
+        $this->assertEquals($this->source->getName(), SourceTest::TEST_STRING);
+    }
+
+    public function testEmail()
+    {
+        $this->source->setEmail(SourceTest::TEST_STRING);
+        $this->assertEquals($this->source->getEmail(), SourceTest::TEST_STRING);
+    }
+
+    public function testContactName()
+    {
+        $this->source->setContactName(SourceTest::TEST_STRING);
+        $this->assertEquals($this->source->getContactName(), SourceTest::TEST_STRING);
+    }
+
+    public function testIsActive()
+    {
+        $this->source->setIsActive(SourceTest::TEST_IS_ACTIVE);
+        $this->assertEquals($this->source->getIsActive(), SourceTest::TEST_IS_ACTIVE);
+    }
+
+    public function testDescription()
+    {
+        $this->source->setDescription(SourceTest::TEST_STRING);
+        $this->assertEquals($this->source->getDescription(), SourceTest::TEST_STRING);
+    }
+
+    public function testLatitude()
+    {
+        $this->source->setLatitude(SourceTest::TEST_STRING);
+        $this->assertEquals($this->source->getLatitude(), SourceTest::TEST_STRING);
+    }
+
+    public function testCountryId()
+    {
+        $this->source->setCountryId(SourceTest::TEST_ID);
+        $this->assertEquals($this->source->getCountryId(), SourceTest::TEST_ID);
```

Comment:
I have the same opinion on this topic, but is it ok to keep them? They dont't harm and add up the testcoverage, as they are already there.